assets/src/js/store-lists.js

```diff
@@ -37,6 +37,12 @@
             // Submit the form
             $( '#dokan-store-listing-filter-form-wrap .apply-filter #apply-filter-btn' ).on( 'click', this.submitForm );
 
+            if ( ! this.getQueryParam( 'latitude' ) && ! this.getQueryParam( 'longitude' ) ) {
+                $.when( $( '#dokan-store-listing-filter-wrap .dokan-store-list-filter-button, #dokan-store-listing-filter-wrap .dokan-icons, #dokan-store-listing-filter-form-wrap .apply-filter #cancel-filter-btn ' ).trigger( 'click' ) ).then( function() {
```

Comment:
Why do we need this long query selector for this?
`#dokan-store-listing-filter-wrap .dokan-store-list-filter-button, #dokan-store-listing-filter-wrap .dokan-icons, #dokan-store-listing-filter-form-wrap .apply-filter #cancel-filter-btn`